circle.yml

```diff
@@ -19,10 +17,11 @@ test:
 
           case $CIRCLE_NODE_INDEX in
           0)
-            ./gradlew build
+            ./gradlew build --parallel
             ;;
           1)
-            ./gradlew testBouncyCastle testSun
+            ./gradlew generateLock verifyDependencyLocksAreCurrent --stacktrace --parallel
```

Comment:
looking at the circle timing this appears to only take 11s, in which case I'd prefer to put it in the dependencies block and just not continue with the rest of the build